core/tests/test_utils_test.py

```diff
@@ -53,14 +53,14 @@ def test_enable_feature_flags_decorator(self) -> None:
 
     @test_utils.enable_feature_flags([
         feature_flag_list.FeatureNames.DUMMY_FEATURE_FLAG_FOR_E2E_TESTS,
-        feature_flag_list.FeatureNames.DIAGNOSTIC_TEST
```

Comment:
This functionality is only testing whether we can successfully enable multiple feature flags or not. Since we are deprecating the diagnostic_test feature flag, I'm replacing it with any other feature flag.